projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -448,63 +448,149 @@
-    // by a pixel due to improper browser rounding; browsers aren't rendering on the
-    // sub-pixel required but rounding up to the nearest whole pixel...
-    font-size: variables.$clr_baselineRem_0_5 - variables.$clr_baselineRem_1px;
-    letter-spacing: normal;
     @include mixins.css-var(
-      line-height,
-      clr-alert-sm-line-height,
-      variables.$clr_baselineRem_0_667,
+      font-size,
+      clr-alert-sm-font-size,
+      // need to use 11 instead of 12 here or else vertical alignment is thrown off
+      // by a pixel due to improper browser rounding; browsers aren't rendering on the
+      // sub-pixel required but rounding up to the nearest whole pixel...
+      variables.$clr_baselineRem_0_5 - variables.$clr_baselineRem_1px,
       variables.$clr-use-custom-properties
     );
+    @include mixins.css-var(font-weight, clr-alert-sm-font-weight, 400, variables.$clr-use-custom-properties);
+    @include mixins.css-var(letter-spacing, clr-alert-sm-letter-spacing, normal, variables.$clr-use-custom-properties);
 
     .alert-items {
-      padding: ($alertSmallNudge - alert-variables.$clr-alert-borderwidth)
-        (variables.$clr_baselineRem_0_25 - alert-variables.$clr-alert-borderwidth);
+      @include mixins.css-var(
+        padding,
+        clr-alert-sm-padding,
+        ($alertSmallNudge - alert-variables.$clr-alert-borderwidth)
+          (variables.$clr_baselineRem_0_25 - alert-variables.$clr-alert-borderwidth),
+        variables.$clr-use-custom-properties
+      );
     }
 
     .alert-item {
-      padding-top: variables.$clr_baselineRem_1px;
-      margin-bottom: $alertSmallNudge;
+      @include mixins.css-var(
+        padding-top,
+        clr-alert-sm-item-padding-top,
+        variables.$clr_baselineRem_1px,
+        variables.$clr-use-custom-properties
+      );
+
+      @include mixins.css-var(
+        margin-bottom,
+        clr-alert-sm-item-padding-top,
+        $alertSmallNudge,
+        variables.$clr-use-custom-properties
+      );
+
+      // ATM
+      // remove
+      @include mixins.css-var(
+        align-items,
+        clr-alert-sm-item-text-align-items,
+        normal,
+        variables.$clr-use-custom-properties
+      );
 
       &:last-child {
         margin-bottom: 0;
       }
     }
 
+    // ATM
+    // remove
+    &.alert-app-level {
+      .alert-item {
+        @include mixins.css-var(
+          align-items,
+          clr-alert-sm-item-text-align-items,
+          center,
+          variables.$clr-use-custom-properties
+        );
+      }
+    }
+
     .alert-icon-wrapper {
       padding-top: 0;
-      height: variables.$clr_baselineRem_0_667;
+      @include mixins.css-var(flex-basis, clr-alert-sm-icon-size, 1.25rem, variables.$clr-use-custom-properties);
+      @include mixins.css-var(
+        height,
+        clr-alert-sm-icon-wrapper-height,
+        variables.$clr_baselineRem_0_667,
+        variables.$clr-use-custom-properties
+      );
```

Comment:
The wrapper gets displaced from the icon inside:
![Screenshot 2023-08-21 at 20 28 00](https://github.com/vmware-clarity/ng-clarity/assets/127101685/447c5d6f-0850-4c62-a184-2cbec90662bf)
We can add `display: flex, align-self: center` to align the wrapper vertically to the center

Reply:
This change is added in commit 3b37d84